README.md

```diff
@@ -101,15 +101,36 @@ web3.config.defaultBlock = "latest"
 
 ### Timeouts, blocking and nonblocking requests
 
-
-All function and property requests block until a respose is received.  Asynchronous function calling has not yet been implemented.
+All function and property requests block until a response is received.  Asynchronous function calling has not yet been implemented.
 
 ```python
 # Blocks indefinitely
 >>> web3.eth.getBalance("0xaddress", timeout=None)
 23423234
 ```
 
+### Not implemented functionality
+
+web3.py does not implement all JavaScript API yet. In the case of missing functionality you can directly call the underlying RPC API using [ethereum-rpc-client](https://github.com/pipermerriam/ethereum-rpc-client). For example you can use this to access the filter logs for events (topics) of smart contracts.
```

Comment:
I would prefer to route people to just directly use `web3.request_manager.request_blocking("eth_mining", [])` as I don't think I plan on supporting the `ethereum-rpc-client` and `ethereum-ipc-client` libraries going forward.